media_player.py

```diff
@@ -7,12 +7,14 @@
 import requests
 
 from homeassistant.components.media_player import (
-    MEDIA_PLAYER_SCHEMA, MEDIA_TYPE_MUSIC, MEDIA_TYPE_CHANNEL, MEDIA_TYPE_PLAYLIST, PLATFORM_SCHEMA,
+MediaPlayerDevice, PLATFORM_SCHEMA)
```

Comment:
If you keep `MEDIA_PLAYER_SCHEMA` in here then we don't need to change it below

Reply:
If I import MEDIA_PLAYER_SCHEMA, I get this error:

```
INFO:homeassistant.util.package:Attempting install of colorlog==4.0.2
Testing configuration at /config
Failed config
  General Errors: 
    - Platform not found: media_player.rn301

Successful config (partial)
```
What's the reason for that I don't know